### DISCUSSION TITLE
Contributing On Windows

### DISCUSSION THREAD

**Discussion by chenyair** [ubuntu-flutter-community/settings#64] [Q&A] [answered]
Hello 😀
I cloned the project and I saw there is only a linux direcotry.
Does it mean I can't contribute if I'm developing on a windows pc?
Thanks for the help!

**Comment by Feichtmeier** [comment, marked as answer]
Hi 👋 
Yes, in this very rare case you really need a Linux desktop. Actually flutter is cross plattform, but this app uses specific linux desktop only APIs to modify the underlying settings database, bluetooth (WIP) and wifi (WIP).
Best regards :) And thanks for your wish to contribute

**Reply by chenyair** [reply]
Alright thank you for your time

**Comment by jpnurmi** [comment]
It should be possible to run the UI in WSL2, but it's probably far from the ideal development environment because the sole purpose of this app is to integrate with the Linux system settings. :slightly_smiling_face:
